01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,13 @@
+numbers = 1..20
+numbers.each do |i|
```

Comment:
最早ローカル変数を使う必要もなくなったと思うので、

```suggestion
(1..20).each do |i|
```

でよいかと思います。

Reply:
なるほど、確かにそうですね。。
指摘いただきありがとうございます。修正しました。